### PR TITLE
HW3 O_Bilynska

### PR DESCRIPTION
**Pull request by Bilunska**

### CODE REVIEW

File: src/hw2/Comics.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package hw2;
+
+public class Comics extends Library implements Artistic {
+    private String current_author;
+    private int current_year;
```
Comment (olobanok) [inline src/hw2/Comics.java:5]: according to conventions fields and methods should be named in camelcase

File: src/hw2/Admin.java [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package hw2;
+
+class Admin extends Comics {
```
Comment (olobanok) [inline src/hw2/Admin.java:3]: why does your Admin extends Comics? 
does it represent IS-A relationship?

File: src/hw2/Admin.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+    private int id;
+
+    Admin(String a, int b, int i) {
```
Comment (olobanok) [inline src/hw2/Admin.java:6]: don't use single letter names for your method parameters, they are not informative

File: src/hw2/Comics.java (hunk 1 above)
Comment (MychkaO) [reply src/hw2/Comics.java:5]: Thanks. In HW3 I resolve all about code convention.

File: src/hw2/Admin.java (hunk 2 above)
Comment (MychkaO) [reply src/hw2/Admin.java:3]: I wanted that Admin has access to current book, which Admin need to find for current User. 
Admin a1 = new Admin(batman.currentAuthor(), batman.currentYear(), 1);
So it seems like HAS-A. Need to think about this in HW3.

**Pull request closed without merging**